lib/app-tray.js

```diff
@@ -18,19 +19,15 @@ function truncateStr(str, len) {
   return str;
 }
 
+// Set-up for OS
+assets = assets[process.platform] || assets.any;
+
 // Define a function to set up our tray icon
 exports.init = function (gme) {
   // Set up our tray
   var trayMenu = new Menu();
   trayMenu.append(new MenuItem({
-    label: 'Show/hide window',
-    click: gme.onTrayClick
-  }));
-  trayMenu.append(new MenuItem({
-    type: 'separator'
-  }));
-  trayMenu.append(new MenuItem({
-    label: 'Play/Pause',
+    label: 'Play / Pause',
```

Comment:
Yea, I would like to use the proper status -- there's even probably a deleted TODO or log message about it (or maybe in `google-music-webkit`). However, for now I would like to keep it as is